web/src/js/backends/websocket.tsx

```diff
@@ -94,6 +95,19 @@ export default class WebsocketBackend {
         }
     }
 
+    sendMessage(resource, data) {
```

Comment:
Maybe we should add a dedicated `updateFilter(name, expression)` here? This way the actual WebSocket message format is contained in the WebSocket backend and does not leak into the rest of the codebase.